joplin/templates/wagtailadmin/pages/search.html

```diff
@@ -0,0 +1,24 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n %}
+{% block titletag %}{% trans 'Search' %}{% endblock %}
+{% block extra_js %}
+  {{ block.super }}
+
+  <script>
+      window.headerSearch = {
+          url: "{% url 'wagtailadmin_pages:search' %}",
+          termInput: "#id_q",
+          targetOutput: "#page-results"
+      }
+  </script>
+{% endblock %}
+
+{% block content %}
+  {% trans "Search" as search_str %}
+  {% include "wagtailadmin/shared/header.html" with title=search_str search_url="wagtailadmin_pages:search" icon="search" %}
+
+  <div id="page-results">
+      {% include "wagtailadmin/pages/search_results.html" with query_string=query_string %}
+  </div>
```

Comment:
Here is where can control what variables we can pass from the `joplin_search_views.js` to `search_results.html`
- We needed to pull this up from the wagtail root to modify this transation.